docs/spec-files/transports.md

```diff
@@ -15,6 +15,48 @@ Each transport MUST define:
 - how [IANA media types](#iana-media-types) of the content are provided, e.g., through `Content-Type` header, etc.
 - where additional context definition is hosted, e.g., in case the `serviceEndpoint`  object has extra properties specific to the transport.
 
+
+### Agent Constraint Disclosure
+
+As mentioned above, DIDComm Messaging is designed to be transport-independent.  Given the wide varity of transports that can be conceived, some agents may have additional constraints that they would like to disclose regarding how they communicate.  These generic and customizable constraints may vary over time or be unique to special categories of agents and may be expressed using using the [Discover Features Protocol](discover-features-protocol-10). The following includes a subset of the many different agent constraints that can be expressed:
+
+#### max\_receive\_bytes
+The _optional_ `max_receive_bytes` constraint is used to specify the total length of the DIDComm header plus the size of the message payload that an agent is willing to receive. While the core DIDComm protocol itself does not impose a specific maximum size for DIDComm messages, a particular agent may have specific requirements that necessitate only receiving messages up to a specific length.  For example, IoT devices that may not have the bandwidth or buffering capabilites to accept large messages may choose to only accept _small_ messages.  The definition of _large_ vs _small_ is subjective and may be defined according to the needs of each implementing agent.
+
+When a `max_receive_bytes` capability is specified, any received message that exceeds the agent's stated maximum will be discarded. It is highly recommended that a response message be returned in order to inform the sender why their message was rejected (e.g., exceeding the max size limitation).  The error message to be returned should conform to the DIDComm [Problem Codes](#problem-codes) formatting and should be named as `me.res.storage.message_too_big`.  Using the [Problem Codes](#problem-codes) naming convention will facilitate using the standard DIDComm troubleshooting techniques, such as [Route Tracing](#route-tracing).
```

Comment:
"capability is specified" --> "constraint is specified"

Reply:
@dhh1128 thanks for the wording tweaks, they really help clarify.  I just uploaded the updates.  How do they look?